analytics.js

```diff
@@ -529,7 +535,11 @@ class Analytics {
     } else {
       rudderElement.setProperty({});
     }
-
+    Object.keys(properties).forEach((property) => {
+      if (reservedPropKeywords.indexOf(property) >= 0) {
+        console.error(`Warning! : Reserved keyword used --> ${property} with track call for event --> ${event}`);
```

Comment:
logger.error should be fine I think
other levels probably needs a wrapper/polyfill for support in older browsers